lorien/InfiniteCanvas/Cursor/CursorClass.gd

```diff
@@ -0,0 +1,29 @@
+class_name CursorClass
+extends Node2D
+
+var _sprite : Sprite
+var _brush_size: int
+var _pressure := 1.0
+
+# --------------------------------------------
+func _ready():
+	pass # Replace with function body.
+
+func _draw():
+	pass
+
+func _input(event):
+	pass
+
+func _process(delta):
+	pass
+
```

Comment:
My bad - I thought it would be necessary to add some shared custom logics, but at the end of the PR it wasn't 😛

Reply:
Since some cursors like the MoveCursor have a Sprite instead of drawing the actual cursor on the Node2D, would make sense to you to make the `BaseCursor` inherit the `Sprite` class?
So it can automatically display a Sprite if a texture is given, else the cursor logic will just draw everything added inside the _draw function.

Reply:
Yes, that sounds like a good idea